tests/setup_nodes.py

```diff
@@ -583,7 +600,7 @@ async def setup_full_system(
     shared_b_tools: BlockTools,
     b_tools: BlockTools = None,
     b_tools_1: BlockTools = None,
-    connect_to_daemon=False,
```

Comment:
This was a bool that meant "read the daemon_port from the config for this BlockTools, and attempt to connect the services to it"